packages/account/src/Styles/account.scss

```diff
@@ -1781,7 +1781,7 @@ $MIN_HEIGHT_FLOATING: calc(
         padding: 3.2rem;
         width: 440px;
 
-        @include mobile-or-tablet-screen {
+        @include mobile-screen {
```

Comment:
why did we remove tablet screen from here

Reply:
modal design should look more like desktop than mobile, that is why 😎